src/lib.rs

```diff
@@ -35,6 +35,32 @@
 
 use core::mem::size_of;
 
+/// Error types for conversions
+#[derive(Clone, Debug, PartialEq, Eq)]
```

Comment:
What about `Copy`?

Reply:
After that thing we had with `ThreadRng` I won't make public APIs `Copy` without a good reason and here there isn't one.